src/main/java/views/DashboardForm.java

```diff
@@ -0,0 +1,253 @@
+package views;
+
+import classes.Settings;
+import classes.SingleClickCopy;
+import classes.Snippet;
+import classes.State;
+import globals.Colors;
+import globals.Globals;
+import views.listCellRenderers.RecentlyAddedCellRenderer;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyVetoException;
+import java.beans.VetoableChangeListener;
+import java.io.FileNotFoundException;
+
+public class DashboardForm {
+    public JPanel mainPanel;
+    public JPanel headerPannel;
+    public JPanel dashboardPannel;
+    public JPanel buttonWrapper;
+    public JButton addSnippetButton;
+    public JButton settingsButton;
+    public JButton allSnippetsButton;
+    public JLabel applicationName;
+    public JPanel listWrapper;
+    public JList<Snippet> recentlyAddedListView;
+    public JPanel recentlyAddedLabelHolder;
+    public JLabel recentlyAddedLabel;
+
+    public DefaultListModel<Snippet> recentlyAddedListModel;
+
+    Settings settings = Globals.settingsParser.getSettings();
+
+    public DashboardForm(){
+
+        headerPannel.setBackground(settings.getColorTheme().getHeaderBackgroundColor());
+        dashboardPannel.setBackground(settings.getColorTheme().getBackgroundColor());
+
+        recentlyAddedListView.setBackground(settings.getColorTheme().getPanelBackgroundColor());
+        applicationName.setText(Globals.APPLICATION_NAME);
+        applicationName.setForeground(settings.getColorTheme().getHeaderTextColor());
+
+        recentlyAddedListModel = new DefaultListModel<>();
+
+        setupDashboard();
+    }
+
+    public void setupDashboard(){
```

Comment:
This does many things, perhaps consider putting it into different functions. A good heuristic is: if you can not describe what it does in one sentence, it probably does too much.

Reply:
Never heard of this heuristic, i'll start using it.